src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -640,7 +640,7 @@ def create_arm_client_factory(credentials):
             from azure.cli.core._debug import change_ssl_cert_verification
             client_type = get_client_class(ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS)
             api_version = get_api_version(cli_ctx, ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS)
-            return change_ssl_cert_verification(client_type(credentials, api_version=api_version,
+            return change_ssl_cert_verification(client_type(credentials, api_version=str(api_version),
```

Comment:
I don't know why is that, but doesn't seem a good idea. `get_api_version` should return a string. If the type is `_ApiVersions`, then casting to string is bad anyway. Or I miss something.